underfs/hdfs/src/main/java/alluxio/underfs/hdfs/HdfsUnderFileSystem.java

```diff
@@ -218,6 +226,54 @@ public boolean exists(String path) throws IOException {
     return hdfs.exists(new Path(path));
   }
 
+  @Override
+  public AccessControlList getAcl(String path) throws UnimplementedException, IOException {
+    FileSystem hdfs = getFs();
+    // TODO(chen): for hadoop <= 2.3, ACL is not supported, there is no getAclStatus method.
```

Comment:
@calvinjia do you have suggestions on a preferred way to handle different behavior for different hadoop versions? I saw there are annotations like `#if hadoop1` `#else`, but I'm not sure whether I can write expressions like `#if hadoop2.3`?